theory/README.md

```diff
@@ -21,6 +23,9 @@ both `antelope` and `antelopes`.)
 
     boat
 
+  ## CODE FOR ANTELOPES ROCK OUT: ```/antelopes?\srocks?\sout/g```
+  ## CODE FOR GOAT/ MOAT: ```/[gm]oat/g```
```

Comment:
`[gm]oat` will only match the first instance - goat but will ignore moat. A capture group will work for our use case:

```
(g|m)oat
```